openedx/features/edly/utils.py

```diff
@@ -333,3 +334,47 @@ def create_learner_link_with_permission_groups(user):
         user.groups.add(new_group)
 
     return user
+
+
+def get_current_site_invalid_certificate_context_data(default_html_certificate_configuration):
+    """
+    Gets current site's context data for invalid certificate.
+
+    Try to get current site's context data for invalid certificate from site configuration
+    or fallback to default html certificate configurations.
+
+    Arguments:
+        default_html_certificate_configuration (dict): Default html configurations dict.
+
+    Returns:
+        dict: Context data.
+    """
+    context = {}
+
+    for key, value in default_html_certificate_configuration.get('default').items():
+        context[key] = value
+
+    current_site_configuration = get_current_site_configuration()
+
+    if not current_site_configuration:
+        return context
+
+    context['platform_name'] = current_site_configuration.get_value('platform_name', settings.PLATFORM_NAME)
+
+    branding = current_site_configuration.get_value('BRANDING', {})
+    context['logo_src'] = branding.get(
+        'logo',
+        default_html_certificate_configuration.get('default', {}).get('logo_src')
+    )
+
+    site_domain = current_site_configuration.get_value('site_domain')
+
+    context['logo_url'] = site_domain if site_domain else default_html_certificate_configuration.get('default', {}).get('logo_url')
+
+    context['company_privacy_url'] = '{}/privacy'.format(site_domain) \
+        if site_domain else default_html_certificate_configuration.get('default', {}).get('company_privacy_url')
```

Comment:
similarly all these lines can be broken into smaller lines. That is better for readability as well

Reply:
Done